pkg/util/binding.go

```diff
@@ -45,3 +47,13 @@ func GetSumOfReplicas(clusters []workv1alpha1.TargetCluster) int32 {
 	}
 	return replicasSum
 }
+
+// GetClusterSetString will get the sets.String from binding's clusters
```

Comment:
```suggestion
// GetClusterSetString will get the sets.String from a cluster slice.
```

Reply:
I modify func name to `ConvertToClusterNames`, and modify comment to:
```
// ConvertToClusterNames will convert a cluster slice to clusterName's sets.String
```